ChangeLog

```diff
@@ -1,3 +1,11 @@
+Changes in squid-5.6 (01 Jun 2022):
+
+	- Bug 5208: Part 1: Restart kids killed by SIGKILL
+	- Fix SQUID-MIB smilint errors
+	- negotiate_kerberos_auth: Initialise default_keytab
+	- Improve handling of Gopher responses
+	- ... and some semi-automated code polish
+
```

Comment:
I would explicitly mention the added support for Gopher type "w" in the ChangeLog.

Reply:
That commit is part of the generic "Improve handling of Gopher responses" line entry.